docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -126,18 +139,81 @@ However, sometimes some of the Managed Servers are different (for example, suppo
 
 This is done by adding the server to the domain resource and setting its `serverStartPolicy` to `ALWAYS`.
 ```
-   domain:
-     spec:
-       managedServers:
-       - serverName: "cluster1_server1"
-         serverStartPolicy: "ALWAYS"
-       ...
+  kind: Domain
+  metadata:
+    name: domain1
+  spec:
+    managedServers:
+    - serverName: "cluster1_server1"
+      serverStartPolicy: "ALWAYS"
+    ...
 ```
 
 {{% notice note %}}
 The server will count toward the cluster's `replicas` count.  Also, if you configure more than the `replicas` servers count to `ALWAYS`, they will all be started, even though the `replicas` count will be exceeded.
 {{%/ notice %}}
 
+### Shutdown options
+
+The domain resource includes the element `serverPod` that is available under `spec`, `adminServer` and each entry of 
+`clusters` and `managedServers`. The `serverPod` element controls many details of how pods are created for server instances.
+
+The `shutdown` element of `serverPod` controls how servers will be shutdown.  This element has three properties:
+`shutdownType`, `timeoutSeconds`, and `ignoreSessions`.  The `shutdownType` property can be set to either `Graceful` 
+or `Forced` specifying the type of shutdown.  The `timeoutSeconds` property configures how long the server is given to 
+complete shutdown before the server is killed.  The `ignoreSessions` property when "false" allows the shutdown
+process to take longer to give time for any active sessions to complete up to the configured timeout.
+The operator runtime monitors this property but will not restart any server pods solely to adjust the shutdown options.
+Instead, server pods created or restarted because of another property change will be configured to shutdown, at the appropriate
+time, using the shutdown options set when the server pod is created.
+
+#### Shutdown environment variables
+
+The operator runtime configures shutdown behavior with the use of the following environment variables. Customers may
+instead simply configure these environment variables directly.  When a customer configured environment variable is present,
+the operator will not override the environment variable based on the shutdown configuration.
+
+| Environment Variables | Default Value | Supported Values |
+| --- | --- | --- |
+| `SHUTDOWN_FORCED` | true | true or false |
```

Comment:
1) Modify the doc here to note here that forced == false implies gracefull.
2) Consider using an enumerated type for SHUTDOWN type instead of true/false - this (a) would align with the domain resource configuration, and (b) would easily allow more options to be added in the future if more are ever needed...